jobserver/settings.py

```diff
@@ -39,7 +38,7 @@
 
 BASE_URL = env.str("BASE_URL", default="http://localhost:8000")
 
-ALLOWED_HOSTS = [furl(BASE_URL).host]
+ALLOWED_HOSTS = ["*"]
```

Comment:
Are we assuming this is OK because nginx/cloudflare will deal with the host check?  I liked the setting to base url host :woman_shrugging:

Reply:
Yep!  The problem was getting correct the IP to add to this list for the CHECKS file to work correctly.  When a deploy runs the new container is spun up and checks are run against it, the IP for that new container was static in my manual tests but I couldn't find anyway to ask another system for it.  Since all external traffic to the site goes through nginx/dokku this change should be fine 🤞